zyte_spider_templates/spiders/serp.py

```diff
@@ -9,9 +9,41 @@
 
 from ..params import MaxRequestsParam
 from ._google_domains import GoogleDomain
+from ._google_hl import GOOGLE_HL_OPTIONS_WITH_CODE, GoogleHl
 from .base import BaseSpider
 
 
+class GoogleHlParam(BaseModel):
+    hl: Optional[GoogleHl] = Field(
+        title="UI Language",
```

Comment:
Will rename to User Language for consistency with the matching geolocation parameter.